### PR TITLE
Eclipse Plugin Search Command

### LINKED ISSUE #1178: Eclipse Plugin Search Command
https://marketplace.eclipse.org/

### PR DESCRIPTION
**Pull request by almeidx**
Closes #1178

### CODE REVIEW

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -1701,5 +1701,15 @@
...
+  },
+  "eclipseplugin": {
+    "commandDescription": "Search for plugins on the Eclipse Plugin Marketplace",
+    "commandUsage": "<plugin name>",
+    "marketplace": "Eclipse Plugin Marketplace",
+    "by": "by [{{name}}]({{homepage}})",
+    "installs": "{{count}} installs",
+    "lastUpdate": "`{{version}}` was published {{ago}}",
+    "license": "License",
+    "install": "Install using Eclipse Marketplace Client"
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1713]:
```suggestion
    "install": "Drag this link to your Eclipse workspace to install"
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

File: src/locales/en-US/commands.json (hunk 1 above)
Comment (almeidx) [reply src/locales/en-US/commands.json:1713]: Does it actually work?

Comment (pedrofracassi) [reply src/locales/en-US/commands.json:1713]: It's the same link that the website tells you to drag into eclipse, so yeah, it should work.

Comment (metehus) [reply src/locales/en-US/commands.json]: "it should work" famous last words

**Comment by Doges** [comment]
also resolve this one

**Review by Doges — APPROVED** [review]
LGTM

**Comment by pedrofracassi** [comment]
@almeidx conflicts pls

**Comment by almeidx** [comment]
> @almeidx conflicts pls

yessir

**Review by pedrofracassi — APPROVED** [review]
LGTM

**Review by Doges — APPROVED** [review]
LGTM

**Review by metehus — APPROVED** [review]

**Pull request merged**